client/src/components/ProTx/components/charts/ChartDiagram1CBC.js

```diff
@@ -1,75 +1,39 @@
 import React from 'react';
 import Plot from 'react-plotly.js';
 
-class ChartDiagram1CBC extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      data: [
-        {
-          x: [1, 2, 3, 4, 5, 6, 7],
-          y: [1, 6, 3, 11, 4, 9, 13],
-          type: 'scatter',
-          mode: 'lines+markers',
-          marker: { color: 'red' }
-        },
-        { type: 'bar', x: [1, 2, 3, 4, 5, 6, 7], y: [2, 6, 3, 7, 4, 9, 15] }
-      ],
-      layout: {
-        // width: '100%',
-        // height: '100%',
-        autosize: true,
-        title: 'Scatterplot w/Bar Chart Example'
+function ChartDiagram1CBC() {
+  const state = {
+    data: [
+      {
+        x: [1, 2, 3, 4, 5, 6, 7],
+        y: [1, 6, 3, 11, 4, 9, 13],
+        type: 'scatter',
+        mode: 'lines+markers',
+        marker: { color: 'red' }
       },
-      config: {
-        // scrollZoom: true,
-        // staticPlot: true,
-        // displayModeBar: false,
-        displaylogo: false,
-        modeBarButtonsToRemove: [
-          // 'toImage'
-        ],
-        modeBarButtonsToAdd: [
-          // {
-          //   name: 'color toggler',
-          //   icon: icon1,
-          //   click: function(gd) {
-          //     var newColor = colors[Math.floor(3 * Math.random())]
-          //     Plotly.restyle(gd, 'line.color', newColor)
-          //   }
-          // },
-          // {
-          //   name: 'button1',
-          //   icon: Plotly.Icons.pencil,
-          //   direction: 'up',
-          //   click: function(gd) {alert('button1')
-          //   }
-          // }
-        ],
-        // showLink: true,
-        showEditInChartStudio: true,
-        plotlyServerURL: 'https://chart-studio.plotly.com',
-        linkText: 'Edit Chart in Plotly Studio',
-        responsive: true,
-        doubleClickDelay: 1000,
-        useResizeHandler: true
-      }
-    };
-  }
+      { type: 'bar', x: [1, 2, 3, 4, 5, 6, 7], y: [2, 6, 3, 7, 4, 9, 15] }
+    ],
+    layout: {
+      width: '100%',
```

Comment:
@taoteg ,  note the change here from:
```
autosize: true
```
to 
```
width: '100%',
height: '100%',
```

i saw the chart size changing when changing different drop down selections but this did the trick. i assume you saw this as well.  this seems to "fix" it for now till we work on this again